django_extensions/db/models.py

```diff
@@ -41,17 +41,40 @@ class Meta:
         abstract = True
 
 
+class ActivatorQuerySet(models.query.QuerySet):
+    """ ActivatorQuerySet
+    Query set that returns statused results
+    """
+    def active(self):
+        """ Returns active query set """
+        return self.filter(status=ActivatorModel.ACTIVE_STATUS)
+
+    def inactive(self):
+        """ Returns inactive query set """
+        return self.filter(status=ActivatorModel.INACTIVE_STATUS)
+
+
 class ActivatorModelManager(models.Manager):
     """ ActivatorModelManager
     Manager to return instances of ActivatorModel: SomeModel.objects.active() / .inactive()
     """
+    def get_query_set(self):
+        """ Proxy to `get_queryset`, drop this when Django < 1.6 is no longer supported """
+        return self.get_queryset()
+
+    def get_queryset(self):
+        """ Use ActivatorQuerySet for all results """
+        return ActivatorQuerySet(model=self.model, using=self._db)
+
```

Comment:
@chrisforrette 
I think these get_queryset methods are problematic, because Django might use them automatically.

I am not sure, but this might get changed to be a private/local function?

Otherwise: can this be made to call `super` / parent classes (easily)?
Maybe similar to QueryManagerMixin from https://github.com/carljm/django-model-utils/blob/master/model_utils/managers.py#L198, where it also extends `__init__`.

With Django 1.7 the default is:

    def get_queryset(self):
        return self._queryset_class(self.model, using=self._db, hints=self._hints)

I had problems because this `get_queryset` method overwrote the queryset from `objects = BaseManager.from_queryset(BaseQuerySet)()` being used.

I am now using `objects = MyManager.from_queryset(MyQuerySet)()`, where `MyQuerySet` inherits from ActivatorQuerySet.